src/python/grpcio_tests/tests/qps/basic_benchmark_test.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+# Copyright 2019 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This test benchmarks Python client/server.
+set -ex
+
+declare -a DRIVER_PORTS=("10086" "10087")
+SCENARIOS_FILE=src/python/grpcio_tests/tests/qps/scenarios.json
+
+function join { local IFS="$1"; shift; echo "$*"; }
+
+if [[ -e "${SCENARIOS_FILE}" ]]; then
+    echo "Running against scenarios.json:"
```

Comment:
Maybe this could be `echo "Running against ${SCENARIOS_FILE}:"`?

Reply:
That's better!